runtime/stdlib/contracts/test.cdc

```diff
@@ -258,6 +278,20 @@ pub contract Test {
         /// Overrides any existing configuration.
         ///
         pub fun useConfiguration(_ configuration: Configuration)
+
+        /// Returns all the logs from the blockchain, up to the calling point.
+        ///
+        pub fun logs(): [String]
+
+        /// Returns the service account of the blockchain. Can be used to sign
+        /// transactions with this account.
+        ///
+        pub fun serviceAccount(): Account
+
+        /// Returns all events emitted from the blockchain, optionally filtered
+        /// by event type.
+        ///
+        pub fun getEvents(_ eventType: String): [AnyStruct]
```

Comment:
I feel it would be better to have two separate functions for the two cases:
 1) Returning all events. Maybe `events()` with no params
 2) Returning filtered events. Maybe `eventsOfType(_ typeName: String)`

WDYT?

Reply:
Seems better indeed :+1: . I kept the single function in the `BlockchainBackend` interface, and added separate functions in the `Blockchain` struct. This way, developers will use the dedicated functions for the two cases, with one implementation under the hood. I did this to avoid adding boilerplate code. Let me know if that is fine, or we should split it even further down.

Reply:
Sounds good